tuist/tuist
# Escape header search paths for external dependencies from SPM

Resolves <https://github.com/tuist/tuist/issues/6487>

### Short description 📝

I wanted to integrate a Swift Package as external dependency and one target uses a C setting with a header search path containing whitespace. This whitespace isn't properly escaped in Tuist. This fixes this.

### How to test the changes locally 🧐

There's a modification of one unit test that tests this. Alternatively run tuist generate with the sample attached in <https://github.com/tuist/tuist/issues/6487> and verify the header search path is correct.

### Contributor checklist ✅


### Reviewer checklist ✅
